students/slawinski_amadeusz/lesson_06_dicts_tuples_sets_args_kwargs/10_the_number_of_distinct_words_in_some_text.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+lines = int(input())
+
+uniquewords = set()
+
+for i in range(lines):
+    linewords = set(input().split())
```

Comment:
jeżeli wykorzystasz funkcje `update()` to druga linijka w pętli jest zbędna i nie potrzebujesz dodatkowej zmiennej:
`uniquewords.update(input().split())`